mollie/api/objects/base.py

```diff
@@ -2,7 +2,7 @@ class Base(dict):
 
     _resource = None
 
-    def __init__(self, data, resource=None):
+    def __init__(self, data, resource=None, client=None):
```

Comment:
There should be a TODO in the docstring explaining that we we'll remove the `self._resource` later on, when all usage of `self._resource` is refactored to using `self.client`.